src/plugins/intel_npu/src/al/src/sync_infer_request.cpp

```diff
@@ -19,7 +19,7 @@ constexpr size_t BATCH_AXIS = 0;
 
 namespace intel_npu {
 
-SyncInferRequest::SyncInferRequest(const std::shared_ptr<const ICompiledModel>& compiledModel)
+SyncInferRequest::SyncInferRequest(std::shared_ptr<const ICompiledModel> compiledModel)
```

Comment:
Now you need to do "_compiledModel(std::move(compiledModel))"

Reply:
done, thx